dts/msm8226/apq8026-lg-lenok.dts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	compatible = "qcom,apq8026";
```

Comment:
```suggestion
	compatible = "lg,lenok", "qcom,apq8026", "lk2nd,device";
```

assuming you use `lg,lenok` for mainline?

Also, can you add a `model = "LG G Watch R"` or something (and check that it displays on the screen)?